src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="get" 
+      autocomplete="off">
+      <fieldset class="box">
+        <legend>Personal information</legend>
+        <label class="label_form">
+          Surname:
+          <input 
+            type="text"
+            name="surname"
+          >
+        </label>
+        <label class="label_form">
+          Name:
+          <input 
+            type="text"
+            name="username"
+          >
+        </label>
+        <label class="label_form">
+          How old are You?
+          <input 
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            
+          >
+        </label>
+        <label class="label_form">
+          Full date of birth:
+          <input 
+            type="date"
+            name="date"
+          >
+        </label>
+        <label class="label_form">
+          I accept the term of the agreement
+          <input 
+            type="checkbox"
+            name="check"
+          >
+        </label>
+        
+
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>Registration</legend>
+        <label class="label_form">
+          E-mail:
+          <input 
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="label_form">
+          Password:
+          <input 
+            type="password"
+            name="password"
+            minlength="7"
+            maxlength="10"
+          >
+        </label>
+        
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>An interesting fact about you!</legend>
+        <div class="label_form">
+          Do you love cats?
+          <label>
+            
+            <input 
```

Comment:
```suggestion
          <label>
            <input 
```
spaces only for same-level nesting elements